go.mod

```diff
@@ -61,6 +60,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.11.2
 )
 
+require github.com/gofrs/uuid v4.2.0+incompatible
```

Comment:
I think this should be added to the above list without the `+incompatible` part.

Reply:
the `+incompatible` is because it does not follow Go's guidelines for module versioning.